meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -430,7 +432,7 @@ EXTRA_IMAGEDEPENDS:remove:stm32mp15-disco = "sdcard-raw-tools-native"
 WKS_FILE_DEPENDS:append:stm32mp15-disco = " u-boot-default-script"
 PREFERRED_PROVIDER_u-boot-default-script:stm32mp15-disco = "u-boot-ostree-scr-fit"
 WKS_FILE_DEPENDS:remove:stm32mp15-disco = "st-image-bootfs st-image-userfs"
-WKS_FILE:sota:stm32mp15-disco = "sdimage-stm32mp157c-dk2-trusted-sota.wks.in"
+WKS_FILE:sota:stm32mp15-disco = "sdimage-stm32mp157c-dk2-optee-sota.wks.in"
```

Comment:
This wks file will need to be renamed as well.

Reply:
I just forgot to push new wks.in file.